README.md

```diff
@@ -321,6 +334,65 @@ diverse sequences but likely with lower native sequence recovery.
 The default sampling temperature is 1. To optimize for native sequence
 recovery, we recommend sampling with low temperature such as 1e-6.
 
+### Command-line interface <a name="cli"></a>
```

Comment:
Nice, is this autogenerated from running `esm-extract -h`?

Reply:
Yep!